tensorflow/python/kernel_tests/slice_op_test.py

```diff
@@ -146,6 +147,19 @@ def testSingleDimension(self):
         slice_val = self.evaluate(slice_t)
         self.assertAllEqual(slice_val, inp[lo:hi])
 
+  def test3Dimension(self):
+    with self.session() as sess:
+      input_shape = [1, 8, 8, 8, 3]
+      inp = np.random.rand(*input_shape).astype("f")
+      a = constant_op.constant(inp, shape=input_shape, dtype=dtypes.float32)
+
+      filter_shape = [2, 2, 2, 3, 3]
+      filters = np.random.rand(*filter_shape).astype("f")
+      conv_t = nn_ops.conv3d(a, filter=filters, strides=[1, 1, 1, 1, 1],
+                             padding="SAME")
+      slice_t = array_ops.slice(conv_t, [0, 0, 0, 0, 0], [1, 2, 1, 2, 1])
+      result = self.evaluate(slice_t)
```

Comment:
Please add a check if the results are correct.

Reply:
Check code added.